docs/configuration/google-analytics.md

```diff
@@ -1,15 +0,0 @@
----
-layout: default
-title: Google Analytics
-parent: Configuration
-nav_order: 4
```

Comment:
With this now being removed we should probably adjust the nav_order for the pages in the Configuration section.

Reply:
It actually just changes precedence I think. It was already missing #1 I think.